src/tests/stub_libdebug.cc

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+
+#define STUB_API "debug/libdebug.la"
+#include "tests/STUB.h"
+
+#include "debug/Messages.h"
+
+#include "debug/Stream.h"
+Debug::Context::Context(const int, const int) : section(0), level(0), sectionLevel(0) {STUB}
+char *Debug::debugOptions = nullptr;
+char *Debug::cache_log = nullptr;
+int Debug::rotateNumber = 0;
+int Debug::Levels[MAX_DEBUG_SECTIONS] = {};
+int Debug::override_X = 0;
+bool Debug::log_syslog = false;
+void Debug::NameThisHelper(const char *) STUB
+void Debug::NameThisKid(int) STUB
+void Debug::parseOptions(char const *) STUB
+static std::ostringstream nilStream;
+std::ostringstream &Debug::Start(const int, const int) STUB_RETVAL_NOP(nilStream)
```

Comment:
@kinkie, just for my own edification, why did you approve a PR that, AFAICT, silently disables basic debugging support in unit tests (that do not use a full debugging library -- libdebug.la)?

Reply:
Oops, missed that bit